microshift_backup_and_restore/microshift-backup-and-restore.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="microshift-backup-and-restore"]
+= About backing up and restoring application data
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-backup-and-restore
+
+toc::[]
+
+You can manually backup and restore data from applications or workloads running in a {product-title} cluster. If you are using an OSTree system, backup and restore are automated. If you are using a non-OSTree system, then you can manually back up and restore application data.
```

Comment:
The thing is we don't actually focus on the application data. All this backup and restore is only about MicroShift data (i.e. etcd database, certificates, etc.).

If application happens to be an operator which stores the data in CRs - then it works as described.
If that's an application with data in some volumes, then we do not cover that

Reply:
```suggestion
You can manually backup and restore the {product-title} database. If you are using an OSTree system, backup and restore are automated. If you are using a non-OSTree system, then you can manually back up and restore application data.
```